src/clr-angular/data/datagrid/datagrid-action-overflow.ts

```diff
@@ -50,6 +76,21 @@ export class ClrDatagridActionOverflow implements OnDestroy {
     if (boolOpen !== this._open) {
       this._open = boolOpen;
       this.openChanged.emit(boolOpen);
+      if (boolOpen && isPlatformBrowser(this.platformId)) {
+        this.zone.runOutsideAngular(() => {
+          // It emits 2+ onStable events in a synchronous sequence. We need to wait for the last one.
```

Comment:
Have we tested this running on prod mode?

Reply:
Thanks for the suggestion. Tested it - works fine.